book_source/03_topical_pages/09_standalone_tools.Rmd

```diff
@@ -164,23 +169,24 @@ b. Use pip to install the dependencies.
 ```bash
 pip install -r requirements.txt
 ```
-3. **Authenticate GEE**. The GEE API needs to be authenticated using your credentials. The credentials will be stored locally on your system. This can be done by,
+4. **Authenticate GEE**. The GEE API needs to be authenticated using your credentials. The credentials will be stored locally on your system. This can be done by,
 ```bash
 #this will open a browser and ask you to sign in with the Google account registered for GEE
 earthengine authenticate
 ```
+5. **Save the Earthdata credentials (optional)**. If you do not wish to enter your credentials every time you use AppEEARS, you may save your username and password inside a JSON file and then add its path [here](https://github.com/PecanProject/pecan/blob/develop/modules/data.remote/inst/appeears2pecan.py#L63)
 
 #### Usage guide:
 This module contains GEE and other processing functions which retrieve data from a specific dataset and source. For example, `gee2pecan_s2()` downloads bands from Sentinel 2 using GEE and `bands2lai_snap()` uses the downloaded bands to compute LAI using the SNAP algorithm. These functions can be used independently but the recommended way is to use `remote_process()` which is a main function that controls all the individual functions to create an organized way of downloading and handling remote sensing data in PEcAn. 
 
 
 **Input data**: The input data must be the coordinates of the area of interest (point or polygon type) and the name of the site/AOI. These information must be provided in a GeoJSON file.
 
-**Output data**: The output data returned by the functions are in the form of netCDF files.
+**Output data**: The output data returned by the GEE are in the form of netCDF files. When using AppEEARS, output is in the form of netCDF files if the AOI type is a polygon and in the form of csv files if the AOI type is a point.
```

Comment:
Do you mean I should convert csv to netCDF? Also currently gee output files do not follow this naming convention if this is final, I'll update the functions to follow this.

Reply:
No worries, I'm aware that gee output doesn't have this convention because naming convention, formats and DB interaction is the next step

Our load_data functions can work with both csv and netcdf, maybe it is not important that they share the same file format at this point, we can fix it later if it turns out we need them to have the same format